example_local_infresnel.ipynb

```diff
@@ -0,0 +1,682 @@
+      "Fitting spline to DEM...\n",
+      "Done\n",
+      "\n",
+      "Computing 391251 paths...\n"
+     ]
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "dd18c45bb12f411fb720c5d2c2345452",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "  0% |          | 0/391251 paths "
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/ltoney/repos/rtm/rtm/travel_time.py:409: UserWarning: 52.3% of DEM is NaN!\n",
+      "  _diff_path_lens = calculate_paths(\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Done\n",
+      "\n",
+      "(5/5) Station 3E.YIF5.01.CDF\n",
+      "\n",
+      "Loading and projecting DEM...\n",
+      "Done\n",
+      "\n",
+      "DEM spacing = 5.40 m -> profile spacing = 2.70 m\n",
+      "\n",
+      "Checking that DEM contains source and receivers...\n",
+      "Done — 168594 invalid paths will be set to NaN\n",
+      "\n",
+      "Fitting spline to DEM...\n",
+      "Done\n",
+      "\n",
+      "Computing 391251 paths...\n"
+     ]
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "f74ce1648ee14b2e8cb7a55c3697bb85",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "  0% |          | 0/391251 paths "
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/ltoney/repos/rtm/rtm/grid.py:480: RuntimeWarning: invalid value encountered in cast\n",
+      "  st[0].stats.sampling_rate).astype(int)\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Done\n",
+      "Done (elapsed time = 106 s)\n",
+      "Diffracted path lengths saved to diff_path_lens.nc\n",
+      "----------------------\n",
+      "PERFORMING GRID SEARCH\n",
+      "Method = 'sum'\n",
+      "----------------------\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "100% |██████████████████████████████████████████████| 391251/391251 grid points "
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Done (elapsed time = 28.6 s)\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "STACK_METHOD = 'sum'  # Choose either 'sum', 'product', or 'semblance'\n",
+    "TIME_METHOD = 'infresnel'  # Choose either 'celerity' or 'fdtd' or 'infresnel'\n",
+    "TIME_KWARGS = {\n",
+    "    'celerity': 343,\n",
+    "    'dem_file': DEM_FILE, \n",
+    "    'stored_result': 'diff_path_lens.nc',\n",
+    "    'n_jobs': -2,\n",
```

Comment:
It might be nice to note what a values of -2 means. I had to dig into the `infresnel` repo and see that -2 is listed as all cpus in `test_grid.py` only (I think).